airflow/migrations/versions/0058_03afc6b6f902_increase_length_of_fab_ab_view_menu_.py

```diff
@@ -40,57 +38,23 @@
 
 def upgrade():
     """Apply Increase length of ``Flask-AppBuilder`` ``ab_view_menu.name`` column"""
-    conn = op.get_bind()
-    inspector = inspect(conn)
-    tables = inspector.get_table_names()
-
-    if "ab_view_menu" in tables:
-        if conn.dialect.name == "sqlite":
-            op.execute("PRAGMA foreign_keys=off")
-            op.execute(
-                """
-            CREATE TABLE IF NOT EXISTS ab_view_menu_dg_tmp
-            (
-                id INTEGER NOT NULL PRIMARY KEY,
-                name VARCHAR(250) NOT NULL UNIQUE
-            );
```

Comment:
This migration dropped the unique key constraint on the name for SQLite. I had to do it with batch mode since there's now batch mode support